.devcontainer/scripts/start-netbox.sh

```
@@ -10,33 +10,79 @@ fi
+  if [ -n "$OLD_PID" ] && kill -0 "$OLD_PID" 2>/dev/null; then
+    if is_expected_pid "$OLD_PID" "python.*rqworker"; then
+      graceful_kill_pid "$OLD_PID"
+    else
+      echo "⚠️  Skipping stale /tmp/rqworker.pid (PID $OLD_PID is not rqworker)"
+    fi
+  fi
+  rm -f /tmp/rqworker.pid
+fi
+
+# Activate NetBox virtual environment
 source /opt/netbox/venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /opt/netbox/venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)